doc/nrf/protocols/wifi/sniffer_rx_operation.rst

```diff
@@ -0,0 +1,136 @@
+﻿.. _ug_nrf70_developing_raw_ieee_80211_packet_reception:
+
+Raw IEEE 802.11 packet reception using Monitor mode
+###################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF70 Series device supports Monitor mode, that is, listening for and receiving all IEEE 802.11 traffic on a given channel, not restricted to a certain SSID. With Monitor mode raw IEEE 802.11 packets can be received directly at the user application layer.
```

Comment:
```suggestion
The nRF70 Series devices support monitor mode. In monitor mode the nRF70 series devices listen on a configured channel and receive all IEEE 802.11 traffic (i.e. belonging to all BSSIDs operating) on that channel. Traffic belonging to all BSSIDs operating on that channel is received and passed . The captured traffic is then piped to the application layer using raw sockets, after prepending it with some metadata.
```

Reply:
This was changed as per earlier discussion to keep it concise